samples/msal-angular-v2-samples/angular11-sample-app/README.md

```diff
@@ -21,3 +21,7 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 
 ## Additional notes
 - The default interaction type for the sample is redirect. The sample can be configured to use popups by changing the `interactionType` in `app.module.ts` to `InteractionType.Popup`. 
+- **To use sample with Angular Universal SSR**: Please see instructions from the [Angular docs](https://angular.io/guide/universal) on how to install Angular Universal with this existing sample. All lines with references to `window` will also need to be removed. Comments indicating the lines to be removed can be found on these pages:
```

Comment:
Another mitigation would be to add a `typeof window !== "undefined"` check to the relevant lines.

Reply:
Added this to the Angular Universal doc.